client/app/components/App.jsx

```diff
@@ -34,7 +34,12 @@ const App = () => {
       .then((relatedProductsResponse) => {
         putInState.relatedProductIDs = relatedProductsResponse.data;
         return axios.get(`/reviews/meta/${putInState.id}`);
+        //return axios.get(`/products/${putInState.id}/styles`);
       })
+      // .then((productStylesResponse) => {
+      //   putInState.differentStyles = productStylesResponse.data;
+      //   return axios.get(`/reviews/meta/${putInState.id}`);
+      // })
```

Comment:
Why comment all this out?

Reply:
that was before we did our change together, i'll delete it on next pull